### DISCUSSION TITLE
github pages

### DISCUSSION THREAD

**Discussion by sahilladva28** [community/community#30277] [Other Feature Feedback, Questions, & Ideas]
hello 
i can't see github pages
please give me suggestion

**Comment by jfullstackdev** [comment]
just want to know whether you successfully created a Page, if not you can still ask questions at this thread

**Comment by sahilladva28** [comment]
Not yet i am do able to show icons of webpage

**Comment by jfullstackdev** [comment]
you only have one public repo and there is no Env tab, so I guess, you are not yet enabling it at the Repo settings

![image](https://user-images.githubusercontent.com/47092464/186939768-9f00ef83-e029-4b4a-9940-03080e0db33d.png)
